exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/using_a_script_languages_container.ipynb

```diff
@@ -0,0 +1,704 @@
+{
```

Comment:
<p>"For these cases, exaslct stores many logs to identify the problem." -&gt; "... creates many logs ..." or "... stores information in many logs ...". Maybe add a sentence. Here is a list of logs that can be inspected in order to find a cause of the problem.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Fixed the first part. I don't the additional sentence is necessary.</p>